scripts/screenshots.js

```diff
@@ -398,8 +457,9 @@ const main = async () => {
   })
   const { browser, page } = await prepareBrowser({ viewport: parsedViewport })
 
-  await screenshotStyleguide(page, args, config)
-  await screenshotStackExamples(page, args)
+  // await screenshotStyleguide(page, args, config)
```

Comment:
```suggestion
  await screenshotStyleguide(page, args, config)
```

Reply:
Solved in https://github.com/cozy/cozy-ui/pull/1664/commits/a157bbf7eb5e472fc830f03c241d17d4fd85dd16